examples/complete/README.md

```diff
@@ -4,6 +4,25 @@
 ### Create an IAM role to target aws account
 
 You can use this module like as below example. You can allow multiple accounts to assume the role adding them into `allowed_accounts` list. 
```

Comment:
You'd better update this line as well according to the changes made.

Reply:
@kychool Done